MatricesForHomalg/gap/Integers.gi

```diff
@@ -216,4 +216,4 @@ InstallMethod( CreateHomalgTable,
 end );
 
 ## create a globally defined ring of integers
-HOMALG_MATRICES.ZZ := HomalgRingOfIntegers( );
+HOMALG_MATRICES.zz := HomalgRingOfIntegers( );
```

Comment:
Is this change deliberate? If yes, two things:
1. This will break ToricVarieties (just grep for `HOMALG_MATRICES.ZZ` in ToricVarieties_project)
2. For consistency, `HOMALG_MATRICES.QQ` should be renamed, too.

Reply:
> 1. This will break ToricVarieties (just grep for `HOMALG_MATRICES.ZZ` in ToricVarieties_project)

Was not intended. I was in a hurry in the morning and probably made a mistake in the regular expression.

> 2. For consistency, `HOMALG_MATRICES.QQ` should be renamed, too.

See above.

Reply:
Ah, I see, then everything's fine :-) I was a bit worried that you were planning larger changes :D

Reply:
I wanted a minimal change. I am unhappy with these changes.

Reply:
> I am unhappy with these changes.

Me too. The suggestion that we should only use names starting with lowercase letters would make everything much worse to read.... :/